Modules/DataModel/Tests/PocketCastsDataModelTests/FmdbGrdbTests.swift

```diff
@@ -17,13 +14,11 @@ final class FmdbGrdbTests: XCTestCase {
     }()
 
     private func setupDataManagerWithFMDB() throws -> DataManager {
-        dbQueue = try XCTUnwrap(FMDatabaseQueue.newTestDatabase())
-        return DataManager(dbQueue: FMDBQueue(fmdbQueue: dbQueue))
+        DataManager(grdbEnabled: false, databaseFileName: "podcast_testDB.sqlite3")
```

Comment:
With the recent test failures I came to the conclusion that these tests were not testing opening the database connection through `DataManager`.

Instead, we were doing that by using `FMDatabaseQueue.newTestDatabase()`, for example. The issue here is that this method was just a duplicate of what is contained in `DataManager` thus it wasn't testing it.

The way it is now it really goes through the logic we have in `DataManager`.